src/utils/filter.js

```diff
@@ -0,0 +1,54 @@
+import lodash from "lodash";
+
+/**
+ * Check if one path matches regular expression or exact string.
+ * @param {{path: string}} pathObject - Entity or object with path property
+ * @param {Array<{path: string}|{regex: RegExp}>} filterObjects - Filter conditions
+ * @return {boolean}
+ */
+function isPathSupported(pathObject, filterObjects) {
+    return (
+        lodash.find(filterObjects, (filterObj) => {
+            if (filterObj.path) {
+                return filterObj.path === pathObject.path;
+            }
+            if (filterObj.regex) {
+                return filterObj.regex.test(pathObject.path);
+            }
+        }) !== undefined
+    );
+}
+
+/**
+ * Check if _all_ paths in concatenated path match filtering conditions.
+ * @param {{path: string}} pathObject - Path object with concatenated path (multipath)
+ * @param {string} multiPathSeparator - String sequence used for concatenation of paths
+ * @param {Array<{path: string}|{regex: RegExp}>} filterObjects - Filter conditions
+ * @return {boolean}
+ */
+function isMultiPathSupported(pathObject, multiPathSeparator, filterObjects) {
+    const expandedPaths = pathObject.path.split(multiPathSeparator).map((p) => ({ path: p }));
+    return lodash.every(expandedPaths, (obj) => isPathSupported(obj, filterObjects));
```

Comment:
Could be done without lodash (few lines but the same with JS only:

```js
for (const expandedPath of expandedPaths) {
    if(!isPathSupported(expandedPath, filterObjects) return false;
}
retrun true;    
```

Reply:
I used the following now, avoiding lodash and explicit for loops:
```javascript
return expandedPaths.every((expandedPath) => isPathSupported(expandedPath, filterObjects));
```